tutorials/Python/01_BasicTypes/BasicTypes.ipynb

```diff
@@ -619,9 +619,9 @@
    "metadata": {},
    "source": [
     "We can also group strings, integers and floating point values into new objects:\n",
-    "- list, surrounded by square brackets, [ ], succesive entries are separated by ','.  These are used a lot.\n",
-    "- tuple, surrounded by round brackets, ( ), succesive entries are separated by ','. Like a list except that, once specified, its elements can't be changed.\n",
-    "- dictionary, surrounded by curly brackets, { }, succesive entries are separated by ','. These are lists of pairs."
+    "- list, surrounded by square brackets, `[ ]`, succesive entries are separated by `,`.  These are used a lot.\n",
```

Comment:
Note: we should be referring to the notebook section that refers to say `lists` here.